eiger_io/fs_handler.py

```diff
@@ -152,6 +152,7 @@ def __call__(self, seq_id, frame_num=None):
             -------
                 A PIMS FramesSequence of data
         '''
+        self.seq_id = seq_id
```

Comment:
ideally we don't want to save any datum information in the class. We should only need information from the resource, and use the datum to return a piece of the resource